### DISCUSSION TITLE
First Installation/Personal Preset Ideas

### DISCUSSION THREAD

**Discussion by MlerSu** [ppy/osu#25632] [Ideas]
#1. First Installation
I've been kind of annoyed with the fact that Osu! Lazer doesn't ask you where you want to install it PRIOR to when it actually does.

#2. Personal Preset Ideas
I feel like having the option to turn on several mod presets would be a neat idea. In terms of conflicting adjustments (like two presets having different AR/HP values) there could possibly be a popup asking the player as to which they would want to use.

**Comment by Rekunan** [comment]
It might be better to just have it automatically resolve conflicts depending on the order of the presets in the list (ex: higher is higher priority).
